src/main/java/org/mandas/docker/client/DefaultDockerClient.java

```diff
@@ -536,7 +534,7 @@ private String toRegExp(String hostnameWithWildcards) {
 
   @Override
   public String getHost() {
-    return fromNullable(uri.getHost()).or("localhost");
+    return ofNullable(uri.getHost()).orElse("localhost");
```

Comment:
I hadn't used this before and had to read the docs.  Seems pretty cool!  Let's see if I understand.
if `uri.getHost()` is non-null, `ofNullable` returns an Optional containing the value of uri.getHost().
if `uri.getHost()` is null, `ofNullable` returns an empty Optional.
if the invoking Optional has a value, `orElse` returns that value.
if the invoking Optional does not have a value, `orElse` returns "localhost".
Is that right?

Reply:
correct